_data/locales/en.yml

```diff
@@ -395,4 +395,17 @@ Tumbleweed_features:
   gamepad:
     name: Gaming
     description: |
-     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
\ No newline at end of file
+     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
+Leap_ovw_title: A brand new way of building openSUSE and a new type of a hybrid Linux distribution
```

Comment:
We tend to avoid capitalizing strings here

Reply:
done